src/smart/SmartPatient.js

```diff
@@ -101,8 +101,8 @@ export function SmartPatient() {
         console.log(e);
       }
 
-      let epTypeStr = process.env?.REACT_APP_CCSM_EPISODEOFCARE_TYPES ?? 'urn:oid:1.2.840.114350.1.13.284.2.7.4.726668.130|2,urn:oid:1.2.840.114350.1.13.284.3.7.4.726668.130|2';
-      
+      let epTypeStr = process.env?.REACT_APP_CCSM_EPISODEOFCARE_TYPES ?? 'urn:oid:1.2.840.114350.1.13.284.3.7.4.726668.130|2';
```

Comment:
Could you add a quick comment about `REACT_APP_CCSM_EPISODEOFCARE_TYPES` as well as `REACT_APP_CCSM_OBSERVATION_CATEGORIES` in https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/wiki/Environment-Setting-for-Dashboard?